tls/s2n_tls_parameters.h

```diff
@@ -92,24 +92,65 @@
 #define TLS_EXTENSION_SUPPORTED_VERSIONS   43
 #define TLS_EXTENSION_KEY_SHARE            51
 
-/* TLS Signature Algorithms - RFC 5246 7.4.1.4.1*/
+/* TLS Signature Algorithms - RFC 5246 7.4.1.4.1 */
+/* https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-16 */
 #define TLS_SIGNATURE_ALGORITHM_ANONYMOUS   0
 #define TLS_SIGNATURE_ALGORITHM_RSA         1
 #define TLS_SIGNATURE_ALGORITHM_DSA         2
 #define TLS_SIGNATURE_ALGORITHM_ECDSA       3
+#define TLS_SIGNATURE_ALGORITHM_PRIVATE     224
 
 #define TLS_SIGNATURE_ALGORITHM_COUNT       4
 
+/* TLS Hash Algorithm - https://tools.ietf.org/html/rfc5246#section-7.4.1.4.1 */
+/* https://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml#tls-parameters-18 */
 #define TLS_HASH_ALGORITHM_ANONYMOUS        0
 #define TLS_HASH_ALGORITHM_MD5              1
 #define TLS_HASH_ALGORITHM_SHA1             2
 #define TLS_HASH_ALGORITHM_SHA224           3
 #define TLS_HASH_ALGORITHM_SHA256           4
 #define TLS_HASH_ALGORITHM_SHA384           5
 #define TLS_HASH_ALGORITHM_SHA512           6
-
```

Comment:
if you're removing the line between alg and count, did you want to do the same above for signatures?